src/graphql-server/schema.graphql

```diff
@@ -10,6 +10,10 @@ type HappStoreUser {
   name: String
 }
 
+type HhaUser {
```

Comment:
as mentioned, defined the `HhaUser` type, and then used it below as the return type of `Query.hostingUser` and `Mutation.registerHostingUser`

I propose that we should as a rule (which I am currently breaking elsewhere but will fix) call the hc address of the entity`id` in the frontend, and then other hashes and addresses we can call `hash` or `address` as makes sense in the context.